components/contacts/contact-form.js

```diff
@@ -0,0 +1,162 @@
+/* global fetch */
```

Comment:
I think it's better remove to [.eslintrc](https://github.com/keenethics/keenethics/blob/dev/.eslintrc)